src/Encoding.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Devpark\Transfers24;
+
+/**
+ * Class Encoding.
+ */
+class Encoding extends CodeTranslate
+{
+    const ISO     = 'ISO-8859-2';
+    const UTF     = 'UTF-8';
+    const WINDOWS = 'Windows-1250';
+
+    /**
+     * Get encoding.
+     *
+     * @param string $encoding
+     *
+     * @return string
+     */
+    public static function get($encoding = self::ISO)
```

Comment:
I don't see any point to set any default value here

Reply:
Based on https://github.com/devpark/laravel-przelewy24/blob/master/src/Country.php